api/src/main/java/jakarta/resource/AdministeredObjectDefinition.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 Contributors to Eclipse Foundation. All rights reserved.
```

Comment:
The changes in this PR otherwise look good, but it is not correct to remove the original copyright year. The second copyright year can be updated to the current year, but the original copyright year must remain.
```suggestion
 * Copyright (c) 2012, 2025 Contributors to Eclipse Foundation. All rights reserved.
```

Reply:
> I did not remove anything here. I added line for EF. I'm not with the Oracle. I'm not affiliated with the Oracle.

My review comment was not about the switch from Oracle copyright to Eclipse Foundation copyright.
My comment above is that the original copyright year `2012` that was originally the file needs to remain. The suggestion that I have above adds the `2012` back in and does nothing more.
I don't know what the correct rules and guidance on replacing the Oracle copyright with the Eclipse Foundation copyright. Mentioning @ivargrimstad for clarification on that part.

Reply:
> My comment above is that the original copyright year `2012` that was originally the file needs to remain. 

It's there. I didn't remove that line. I added new one.

> The suggestion that I have above adds the `2012` back in 

I didn't remove it so it's hard to add it back.

> I don't know what the correct rules and guidance are on replacing the Oracle copyright with the Eclipse Foundation copyright. Mentioning @ivargrimstad for clarification on that part.

I'm not very fluent in the law. I tried to preserve the (c) held by the Oracle in the year range, and added modification year for EF.

Thanks for checking.

Reply:
It looks like I misread the changes and thought the Oracle copyright was replaced. I am wrong, it was not replaced.